Cloudinary.Test/AsyncMethodsTest.cs

```diff
@@ -271,68 +294,85 @@ public void TestCreateTransformAsync()
             Assert.AreEqual(true, getResult.Strict);
             Assert.AreEqual(false, getResult.Used);
             Assert.AreEqual(1, getResult.Info.Length);
-            Assert.AreEqual(t.Generate(), new Transformation(getResult.Info[0]).Generate());
+            Assert.AreEqual(m_simpleTransformationName, new Transformation(getResult.Info[0]).Generate());
         }
 
         [Test]
         public void TestExplicitAsync()
         {
-            ExplicitParams exp = new ExplicitParams("cloudinary")
+            var cloudinary = "cloudinary";
+            var type = ResourceParameterType.facebook.ToString();
+
+            ExplicitParams exp = new ExplicitParams(cloudinary)
             {
-                EagerTransforms = new List<Transformation>() { new Transformation().Crop("scale").Width(2.0) },
-                Type = "facebook"
+                EagerTransforms = new List<Transformation>() { m_simpleTransformation },
+                Type = type,
+                Tags = m_apiTag
             };
 
             ExplicitResult expResult = m_cloudinary.ExplicitAsync(exp).Result;
+            AddCreatedPublicId(ResourceParameterType.facebook, expResult.PublicId);
 
-            string url = new Url(m_account.Cloud).ResourceType("image").Add("facebook").
-                Transform(new Transformation().Crop("scale").Width(2.0)).
-                Format("png").Version(expResult.Version).BuildUrl("cloudinary");
+            string url = new Url(m_account.Cloud).ResourceType(Api.GetCloudinaryParam(ResourceType.Image)).Add(type).
+                Transform(m_simpleTransformation).
+                Format(FILE_FORMAT_PNG).Version(expResult.Version).BuildUrl(cloudinary);
 
             Assert.AreEqual(url, expResult.Eager[0].Uri.AbsoluteUri);
         }
 
         [Test]
         public void TestSpriteAsync()
         {
+            var publicId1 = GetUniquePublicId(ResourceParameterType.sprite);
+            var publicId2 = GetUniquePublicId(ResourceParameterType.sprite);
+            var publicId3 = GetUniquePublicId(ResourceParameterType.sprite);
+
+            var spriteTag = $"{GetMethodTag()}_ASYNC";
```

Comment:
We’ve decided to remove async, but without overriding GetMethodTag since all async test methods have “Async” suffix. Moreover, override with parent call would break reflection, as we get the name at a certain position in StackTrace, so we'll need to either duplicate code or create additional methods with stack position as a parameter. Doable, but to my mind, Async suffix of methods is enough.